go.mod

```diff
@@ -1,11 +1,12 @@
-module service
+module %CUSTOM_PLUGIN_SERVICE_NAME%
 
-go 1.13
+go 1.14
 
 require (
 	github.com/gorilla/mux v1.7.4
 	github.com/mia-platform/configlib v1.0.0
 	github.com/mia-platform/glogger v1.0.0
 	github.com/sirupsen/logrus v1.6.0
+	github.com/stretchr/testify v1.3.0
 	gotest.tools v2.2.0+incompatible
```

Comment:
Could you remove the old teest with `gotest.tools`? We could remove this dependency

Reply:
done